examples/osggeometry/osggeometry.cpp

```diff
@@ -543,9 +545,10 @@ class MyTransformCallback : public osg::NodeCallback
 
         float               _angular_velocity;
 
-};
-
+}; //class MyTransformCallback
 
+/// Create a quad that sits behind the geometry from createScene()
+/// and shows the image from the OpenGL docs that geometry duplicates.
```

Comment:
I dislike commenting close brackets as it really doesn't help with readability when you have a modern editor as it'll match brackets for you, and takes away from the clarity of the actual functional code that is doing stuff - the important stuff for understanding code.  I review lots of code so clarity is really important.

As for disabling lighting, I think this is personal preference, the example is meant to be about creating osg::Geometry rather than state, other examples exist to manage state so I don't see a particular value in adding stuff for the example to beautify the on screen result from the perspective from one user.